Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -10922,6 +10926,89 @@ a_Player:OpenWindow(Window);
 			},
 			Inherits = "cEntity",
 		},
+		cExpOrb =
+		{
+			Desc = [[
+				This class represents a experience orb. This entity can be spawned by using cWorld:SpawnExperienceOrb.
+				It's also spawned when a monster is killed or a player is mining coal.
+			]],
+			Functions =
+			{
+				GetAge =
```

Comment:
The pickup entity has also a age method and they have both a life time of 5 mins.
Should I add this to the description? Would make sense.